src/services/SendATFReport.ts

```diff
@@ -81,9 +76,13 @@ class SendATFReport {
     const sortDateAsc = (date1: any, date2: any) => {
       const date = new Date(date1.startTime).toISOString();
       const dateToCompare = new Date(date2.startTime).toISOString();
-      if (date > dateToCompare) { return 1; }
-      if (date < dateToCompare) { return -1; }
-      return 0;
+      if (date > dateToCompare) {
+        return 1;
+      }
+      if (date < dateToCompare) {
+        return -1;
+      }
+      return (date > dateToCompare) ? 1 : (date < dateToCompare) ? -1 : 0;
```

Comment:
This is wrong. Three scenarios: date is after, date is equal, date is before. Hence 1, 0, -1.
Please revert

Reply:
Reverted